sys/include/ztimer.h

```diff
@@ -465,6 +465,42 @@ static inline ztimer_now_t ztimer_now(ztimer_clock_t *clock)
     }
 }
 
+/**
+ * @brief   Get the time until a timer will trigger
+ *
+ * @param[in]   clock          ztimer clock to operate on
+ * @param[in]   timer          ztimer to messuers to
+ *
```

Comment:
please add some note that this is `O(n)` with `n` being the number of active timers on that clock.